script/load_templates

```diff
@@ -141,6 +141,25 @@ sub post_entry{
 
     my %param;
 
+    if ($table eq 'JobTemplates') {
+        unless (defined($entry->{prio})) {
+            # we have to migrate the prio from the TestSuite to the JobTemplate
+            for my $ts (@{$info->{TestSuites}}) {
+                if ($ts->{name} eq $entry->{test_suite}{name}) {
+                    $entry->{prio} = $ts->{prio};
+                }
+            }
+        }
+        unless (defined($entry->{group_name})) {
```

Comment:
I was thinking, when JobGroups::name is 'is_nullable' 0, there should be `unless ($entry->{group_name})`. Since you are validating text user input so '' should be considered invalid.